### ISSUE TITLE
Pulse demons are feeble

### ISSUE THREAD

**Issue by Hinaichigo** [d3athrow/vgstation13#35083]
Recently when I get pulse demon it usually ends with me moving around to try to find APCs because each one barely boosts your max charge, someone then spots me, and they call the engineers who cut the power stationwide and then I just die.
Maybe I'm missing something, but it seems like there's not much you can do to survive once you get discovered (which is easy due to the glow and sparking and sounds), and the power gets shut off.
I think you should be able to survive inside of APCs and SMES and things and survive off the energy stored in them, even if the power supply is cut for that room. If I recall... you used to be able to do something like this. But currently you just start dying once the room has its power cut off and you can't save yourself by hiding in a battery.

Also the max charge increase per APC seems kind of unsatisfying.. it's so miniscule, and it seems like can't really recharge your power from an APC you've already hacked. And its difficult to afford any upgrades as a result.

I think the changes in #33690 resulted in this state of things, as I remember pulse demons seemed a lot more powerful and easier to get going before that.

inb4 skill issue

[discussion][gameplay]

**Comment by DorainGrey** [comment]
Pulse demon is a minor antag, it's not supposed to be especially resilient or strong.

Anyway the previous behaviour was broken, you could not survive off the battery in an APC, instead what happened was you stopped consuming power & could stay in an APC for infinite time hiding which meant that demons never went away.
If you fixed this to consume battery, the difference would be negligible considering how quickly a demon drains an APC anyway.

Pulse Demons are glass cannons, they can cause mass damage, fairly easily if they get a bit of space to grow but are also easy to defeat in the early stages.

**Comment by pedr69** [comment]
yeah theres a couple tricks to pulsedemon that most people don't know because the thing is so fucking obscure
with the malf explode being one SMES and 20 APC's away you absolutely can demolish the station

**Comment by Hinaichigo** [comment]
Well you can always kill them very easily with an ion rifle. 
But.. as soon as you're discovered they can just turn off the power and you're dead unless you managed to get self sustaining charge before that but that requires a massive amount of hacking to get in the first place.

**Comment by pedr69** [comment]
very easily citation needed citation needed - ion bolts pass over the pulsedemon, so if he's not next to a wall or something he's impossible to hit
with a cauldron and large bluespace beakers of iron and uranium, yeah you could probably kill em

**Comment by Killette2** [comment]
wouldn't know I get this rolled less than blob. pulse demon day of the week when?
